xml/ses_glossary.xml

```diff
@@ -22,59 +22,53 @@
  </info>
  <glossdiv>
   <title>General</title>
-  <glossentry>
-   <glossterm>Admin node</glossterm>
+  <glossentry><glossterm>Admin node</glossterm>
    <glossdef>
     <para>
      The node from which you run the <command>ceph-deploy</command> utility to
      deploy &ceph; on OSD nodes.
     </para>
    </glossdef>
   </glossentry>
-  <glossentry>
-   <glossterm>Bucket</glossterm>
+  <glossentry><glossterm>Bucket</glossterm>
    <glossdef>
     <para>
      A point which aggregates other nodes into a hierarchy of physical
      locations.
```

Comment:
Since we're here, a note mentioning that this is different from S3 Buckets or Containers which are akin to folders.

Reply:
Included